src/js/models/interfaces/index.ts

```diff
@@ -14,6 +14,7 @@ export type Peak = {
   latitude: number;
   longitude: number;
   completed: boolean;
+  completionTime: number;
```

Comment:
completionTime powinien być opcjonalny?

Reply:
jak ktoś nie zdobył szczytu to zwracane jest 0

Reply:
więc w sumie na backendzie zawsze jest przesyłane

Reply:
Fajny byłby null albo brak pola, ale 0 też może być